pandas/_libs/tslib.pyx

```diff
@@ -266,20 +266,16 @@ def format_array_from_datetime(ndarray[int64_t] values, object tz=None,
         elif basic_format:
 
             dt64_to_dtstruct(val, &dts)
-            res = '%d-%.2d-%.2d %.2d:%.2d:%.2d' % (dts.year,
-                                                   dts.month,
-                                                   dts.day,
-                                                   dts.hour,
-                                                   dts.min,
-                                                   dts.sec)
+            res = (f'{dts.year}-{int(dts.month):02}-{int(dts.day):02} '
```

Comment:
we need the int here? these *should* already be ints

Reply:
You're absolutely right!

line numer 278:
```python
res += f'.{dts.us / 1000:03}'
```

Resulted as a float, and for some reason I thought that this was the case for everything \*facepalm*

any way, just added a second ```/``` so it will result as an int.